specification/ai/HealthInsights/HealthInsights.Common/model.common.fhir.elements.tsp

```diff
@@ -55,7 +55,7 @@ alias value_x = {
 #suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Keep is much as close to FHIR Spec"
 @doc("""
   Base for all elements
-  Based on [FHIR Element](https://www.hl7.org/fhir/datatypes.html#Element)
+  Based on [FHIR Element](https://www.hl7.org/fhir/R4/element.html)
```

Comment:
What's the issue with the urls? cc @asaflevi-ms

Reply:
We have two of the same ref with different URL links. Here's the other:

https://github.com/Azure/azure-rest-api-specs/blob/ee228d2bdd8496ee93aeb3ed56dd7e619ed7399b/specification/ai/HealthInsights/HealthInsights.Common/model.common.fhir.elements.tsp#L73-L77

The link I'm changing in this PR doesn't actually go to the anchor it looks like it intended to and so matching it to the one in Element (which it's pointing to anyway)